packages/apps/src/initSettings.ts

```diff
@@ -24,7 +33,15 @@ function getApiUrl (): string {
 
     assert(url.startsWith('ws://') || url.startsWith('wss://'), 'Non-prefixed ws/wss url');
 
-    return url;
+    return { param: url, type: 'json-rpc' as EndpointType };
```

Comment:
As above, casting should not be needed at all. (Last comment on this, so we should do a search for all, remove it and fix why it breaks... if explicitly needed at a single point, all ok.)